iosdevapp/ViewModel.swift

```diff
@@ -0,0 +1,55 @@
+//
+//  ModelView.swift
+//  iosdevapp
+//
+//  Created by Leonit Ejupi on 07.02.23.
+//
+
+import Foundation
+
+final class ViewModel: ObservableObject {
```

Comment:
For the view model name I suggest being more precise otherwise it will get confusing when you start having multiple view models for different views. What we use in the project is the view name plus 'Model' at the end. In this case `GenerateImageViewModel`.